Rakefile

```diff
@@ -108,7 +108,8 @@ end
 # Runs `test` target for workspace/scheme/destination
 def run_tests(workspace, scheme, destination)
   settings = "GCC_PREPROCESSOR_DEFINITIONS='${inherited} ENCRYPT_DATABASE=1'" unless !ENV["encrypted"]
-  logName = (ENV["encrypted"] ? "Encrypted" : "") + "#{scheme}.log"
+  # if jenkins passed us a log name, use that, otherwise default to "test.log"
+  logName = (ENV["LOG_NAME"] == nil ? "test.log" : ENV["LOG_NAME"])
```

Comment:
I think for local testing using `rake` it would be easier if the default was `#{scheme}.log"` since the default test runs two schemes for macOS and iOS it would be nice to still get two differently named log files out.

Reply:
Good point, see 3fd8515 and afb2cc4